setup.examples/exim/mailwatch-exim.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Script to apply adjustments for exim
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+Webuser="$1"
+
+/etc/init.d/exim4 stop
+
+cp -f "$DIR/etc/default/exim4" /etc/default/exim4
+cp -R "$DIR"/etc/exim4/* /etc/exim4/
```

Comment:
Remove `cp -R "$DIR"/etc/exim4/* /etc/exim4/`

Should be:

```
cp  -f "$DIR"/etc/exim4/mailscanner_acldefs /etc/exim4/.
cp  -f "$DIR"/etc/exim4/hubbed_hosts /etc/exim4/.
cp  -f "$DIR"/etc/exim4/relay_domains /etc/exim4/.
cp  -f "$DIR"/etc/exim4/00_mailscanner_listmacrosdefs /etc/exim4/conf.d/main/.
cp  -f "$DIR"/etc/exim4/01_mailscanner_config /etc/exim4/conf.d/main/.
```

And to simplify again, remove all these not essential files:

```
host_local_deny_exceptions
local_host_blacklist
local_rcpt_callout
local_sender_blacklist
local_sender_callout
sender_local_deny_exceptions
```

Should be over now.